node-operators/network-node/tanssi/tanssi-node-systemd.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Run a Tanssi Node Using Systemd
+description: Learn how to set up and run a Node for Tanssi networks using Systemd, allowing you to provide API endpoints for applications and users.
+icon: simple-linux
+---
+
+# Run a Tanssi Node Using Systemd
+
+## Introduction {: #introduction }
+
+In this guide, you'll learn how to spin up a Tanssi Node using the latest stable binary file release and manage the service using [Systemd](https://systemd.io){target=\_blank} on Linux systems. Nodes provide essential API endpoints for applications and users to interact with the Tanssi network.
+
+The article follows the good practice of running the service with its own non-root account and granting that account write access to a specific directory. However, you can adapt this article's steps and instructions to your infrastructure configuration, preferences, and security policies.
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+To get started, you'll need access to a computer running an Ubuntu Linux OS with [Landlock](https://docs.kernel.org/security/landlock.html){target=\_blank} enabled and root privileges. You will also need:
+
+- **Node binary files** - an Node requires three binary files: `tanssi-relay`, `tanssi-relay-execute-worker`, and `tanssi-relay-prepare-worker`
+``
+
+The instructions in this guide execute the [latest](https://github.com/moondance-labs/tanssi/releases/latest){target=\_blank} official stable release. However, you can build your own file compiling the [source code](https://github.com/moondance-labs/tanssi){target=\_blank}.
+
+## Check Landlock Support {: #check-landlock }
+
+Tanssi nodes use the Linux kernel's Landlock feature as a security measure to restrict its own access to system resources, limiting the damage if the application is compromised.
+
+Check the Landlock feature support in your system running the following command:
```

Comment:
Tanssi nodes use the Linux kernel's Landlock feature as a security measure to restrict access to system resources, limiting the damage if the application is compromised.

Check the Landlock feature support in your system by running the following command: